assets/dev/scss/editor/panel/controls/_globals.scss

```diff
@@ -153,6 +154,7 @@
 
 	&__preview-item {
 		cursor: pointer;
+		white-space: nowrap;
```

Comment:
Check what currently happens when there is a long color name

Reply:
It wraps